workshop.json

```diff
@@ -109,11 +109,11 @@
             "name": "linux_src",
             "type": "source",
             "source_info": {
-                "url": "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/snapshot/linux-6.11.tar.gz",
-                "filename": "linux.tar.gz",
+                "url": "https://cdn.kernel.org/pub/linux/kernel/v6.x/linux-6.12.31.tar.xz",
+                "filename": "linux.tar.xz",
                 "commands": {
-                    "unpack": "tar -xzf linux.tar.gz",
-                    "get_dir": "tar -tzf linux.tar.gz  | head -n 1",
+                    "unpack": "tar -xf linux.tar.xz",
+                    "get_dir": "tar -tf linux.tar.xz  | head -n 1",
```

Comment:
Hmm, does this need to be `-xJf` and `-tJf`?

Reply:
I test without -J and it works
not sure what this filter does
       -J, --xz
              Filter the archive through xz(1).

Reply:
I guess so